Bio/AlignIO/MafIO.py

```diff
@@ -565,13 +565,15 @@ def get_spliced(self, starts, ends, strand=1):
         """Return a multiple alignment of the exact sequence range provided.
 
         Accepts two lists of start and end positions on target_seqname, representing
-        exons to be spliced in silico.  Returns a *MultipleSeqAlignment* of the
-        desired sequences spliced together.
+        exons to be spliced in silico. A strand may also be provided.
+        Returns a *MultipleSeqAlignment* of the desired sequences spliced together.
+        When strand equals -1, the sequences are reverse complemented before the splice.
```

Comment:
This does not tell me what order to put the starts & ends in. For forward strand it is implicit and obvious enough, but not for the reverse strand.

Perhaps "..., and the start and end ordinates should be in numerically increasing rather than translation order."

Or the opposite, "...,  and the start and end ordinates should be in translation rather than numerically increasing order."

Reply:
I changed the docstring as you suggested, it should be good now.